fastlane/CommonFastfile

```diff
@@ -142,11 +146,43 @@ lane :github_changelog do |options|
       item = items.first
       message = "#{item['title']} (##{item['number']})"
       username = item["user"]["login"]
+      change_types = item["labels"].map do |label|
+        label_name = label["name"]
```

Comment:
If I'm following this correctly, does this mean that if a PR has multiple matching labels, it'll only apply it to the last label found?

Reply:
This is more or less the logic:
- Get all supported labels. If not supported, the label will be equal to "other
- If there are multiple labels:
	- And one is "other". This can happen if it's labeled with a non-supported type, like for example "HOLD", "fix" and "breaking"
		- Remove "other" and leave the other supported types
	- If tagged with "breaking", it will go to the breaking changes section directly since it has preference. For example if "breaking" and "fix", it will go to the breaking changes section
	- If tagged with "feat" and "fix" (this is wrong tagging, it should be either), it will go to the new features section
- If tagged with something that's not "breaking", "fix" or "feat" it just goes directly to "Other changes" since I think having too many sections in the changelog can make it hard to read and most of the types (like "test", "docs", "build") can just be categorized as "Other changes"

What do you think? Some things that I am not sure about are: 
- Should we support multiple labels? I don't think the angular convention supports this (besides breaking + something else) and I think it might be overkill
- Do we enforce not tagging with multiple labels? I think we can do this Danger. Can be an improvement and not necessary for this PR
- Do we add more sections to the changelog?
- Should breaking changes be a section? Or something in the changelog lines, something like:
```
## New Features
- `*BREAKING* Removes StoreKit (#1771) via vegaro (@vegaro)` 
- `Adds StoreKit 3 option (#1772) via vegaro (@vegaro)` 
```
Instead of:

```
## Breaking Changes
- Removes StoreKit (#1771) via vegaro (@vegaro)` 
## New Features
- `Adds StoreKit 3 option (#1772) via vegaro (@vegaro)` 
```

@RevenueCat/core-sdk  I would love your input on this

Reply:
I think what you did makes a lot of sense 👍🏻  especially for now. If we run into other edge cases we can either fix them later, or just move a commit manually as well. This covers the majority of what we need I think :)